internal/app/machined/pkg/controllers/k8s/manifest.go

```diff
@@ -7,9 +7,10 @@ package k8s
 import (
 	"bytes"
 	"context"
+	"encoding/json"
 	"fmt"
-	"html/template"
```

Comment:
This always bites me.

Reply:
I hate this auto-completion thing... it's better if it _asked_ which package I mean